CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+### 0.5.0 - Dec 07, 2020
+
+* Renamed to [hiredis-cluster](https://github.com/Nordix/hiredis-cluster)
+* The C library `hiredis` is an external dependency rather than a builtin part
+  of the cluster client, meaning that `hiredis` v1.0.0 or later can be used.
```

Comment:
Remove the "v". It's not used elsewhere and is not part of the git tag.

Reply:
hiredis uses tags that begins with v, like `v1.0.0`.
Like https://github.com/redis/hiredis/releases/tag/v1.0.0

The hiredis-vip didnt use v-tags, which is a bit weird?

Reply:
Oh... interesting. I didn't actually check the hiredis tags. Should we follow the tradition of hiredis-vip or that of hiredis or should we investigate further? If we should we follow hiredis and release 1.1.0 whenever hiredis is 1.1.0 (if there are new commands, etc.) it could make sense to match hiredis' tag names. Otherwise, I have seen tags without v more often, but I might be biased. I don't have a strong opinion.

Reply:
redis itself dont use v

Reply:
ok, so let's use 0.5.0 without v